public/index.html

```diff
@@ -2411,6 +2411,102 @@ <h3 class="text-lg font-bold text-white bg-dark text-center">AVATARS</h3>
         <span class="text-link">Mini Survey</span>
       </button>
     </section>
+    <!-- Start Eman Section -->
+    <section class="atom m-8">
+      <h2 class="text-3xl border-b-2 pb-3 mb-4">Atoms</h2>
+      <div class="flex justify-between items-center">
+        <button class="bg-primary hoverEffect text-white font-bold px-5 py-2 rounded-full focus:outline-none border flex justify-between items-center">
+            Button Text
+        </button>
+        <button class="border-grey border px-4 py-2 ml-2 rounded-full flex justify-between items-center focus:outline-none">
+          <span class="p-2 rounded-xl bg-primary flex items-center">
+              <span class="p-1 rounded-lg  bg-white"></span>
+          </span>
+          <svg class="text-dark-grey w-4 ml-2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor">
+              <path fill-rule="evenodd" d="M4 3a2 2 0 00-2 2v10a2 2 0 002 2h12a2 2 0 002-2V5a2 2 0 00-2-2H4zm12 12H4l4-8 3 6 2-4 3 6z" clip-rule="evenodd" />
+          </svg>
+          <p class="text-dark-grey pl-2">Label</p>
```

Comment:
still, group tab is not identical with design (Icon , font weight and spacing between them)

Reply:
Done